### DISCUSSION TITLE
How to get details of contributions of different marketing channels at a Geo level.

### DISCUSSION THREAD

**Discussion by chiranjib-acn-cda** [google/meridian#239] [Q&A] [answered]
While developing a model at the geo level, is it also possible to get marketing contributions for marketing channels for different geos? I have gone through the documentation, but I wasn’t able to find any functionality where any insights at geo level can be extracted from the model.
If this insight is available, it can potentially help in understanding the contribution composition of marketing channels across geos.

**Comment by mikewurm** [comment, marked as answer]
The [`Analyzer.media_summary_metrics`](https://developers.google.com/meridian/reference/api/meridian/analysis/analyzer/Analyzer?hl=en#media_summary_metrics) method might be what you are looking for. This method reports results by geo when `aggregate_geos=False`.

You might also be interested in the `incremental_impact`, `roi`, `marginal_roi`, and `response_curves` methods of the Analyzer class.  These methods return the full set of prior/posterior draws for the corresponding metrics, and they all have the`aggregate_geos` argument.

**Reply by chiranjib-acn-cda** [reply]
Thank you for your answer.
